iterators_task_1/student/src/impl.cpp

```diff
@@ -1,2 +1,39 @@
 #include "impl.h"
+#include <vector>
+#include <algorithm>
 
+std::string reverseWords(const std::string& input){
+    std::stringstream sstream(input);
+    std::vector<TokenString> tokens {std::istream_iterator<TokenString>{sstream}, std::istream_iterator<TokenString>{}};
+
+
+    tokens.erase(std::remove_if(tokens.begin(), tokens.end(), [](TokenString token){
```

Comment:
1. Why do you copy TokenString?
2. String stream iterator should automatically skip empty tokens